apps/mobile/components/swipeable-row.tsx

```
@@ -0,0 +1,158 @@
+        style={[
+          styles.actionsContainer,
+          {
+            opacity: actionsOpacity,
+          },
+        ]}
+      >
+        <XStack height="100%">
+          {rightActions.map((action, index) => (
+            <Pressable
+              key={`${action.icon}-${index}`}
+              style={[styles.actionButton, { backgroundColor: action.color }]}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)